doc/operators/start_network.rst

```diff
@@ -9,41 +9,63 @@ Starting a New Network
 
-.. note:: The network certificate should be distributed to users and members to be used as the certificate authority (CA) when establishing a TLS connection with any of the nodes part of the CCF network. When using curl, this is passed as the ``--cacert`` argument.
-
 The certificates and recovery public keys of initial members of the consortium are specified via ``--member-info``. For example, if 3 members should be added to CCF, operators should specify ``--member-info member1_cert.pem,member1_enc_pub.pem``, ``--member-info member2_cert.pem,member2_enc_pub.pem``, ``--member-info member3_cert.pem,member3_enc_pub.pem``.
 
 The :term:`Constitution`, as defined by the initial members, should be passed via the ``--gov-script`` option.
 
-The network is now in its opening state and any new nodes can join the network without being trusted by members.
+The network is now in its opening state and new nodes can join the network. :ref:`members can add other members and users via governance <members/open_network:Opening a Network>`.
+
+Network Identity
+~~~~~~~~~~~~~~~~
+
+The network certificate should be distributed to users and members to be used as the certificate authority (CA) when establishing a TLS connection with any of the nodes part of the CCF network. When using curl, this is passed as the ``--cacert`` argument.
+
+Virtual Mode
+~~~~~~~~~~~~
+
+To start a CCF node in `virtual` mode, operators should run ``$ cchost.virtual --enclave-file /path/to/virtual_enclave_library ...``.
+
+.. warning:: Nodes started in virtual mode provide no security guarantees. They should never be used for production purposes.
+
+PBFT
+~~~~
+
+To use the PBFT consensus protocol, pass ``--consensus pbft``. Please see :ref:`here <developers/consensus:Consensus Protocols>` for more information.
+
+Signature Interval
+~~~~~~~~~~~~~~~~~~
+
+Transaction commit latency in a CCF network is primarily a function of signature frequency. A network emitting signatures more frequently will be able to commit transactions faster,
+but will spend a larger proportion of its execution resources creating and verifying signatures. Setting signature frequency is a trade-off between transaction
+latency and throughput.
 
-.. note:: Once a CCF network is started, :ref:`members can add other members and users via governance <members/open_network:Opening a Network>`.
-.. note:: If starting up the network with PBFT enabled as the consensus protocol, be sure to add the ``--consensus pbft`` CLI argument  when starting up the node. For more info on the provided consensus protocols please see :ref:`here <developers/consensus:Consensus Protocols>`
```

Comment:
That was a lot of notes. As a rule of thumb, I would suggest that a section that contains more than, say, two notes, probably ought to be broken into sub-sections.